### DISCUSSION TITLE
How can I integrate a github template for pull request?

### DISCUSSION THREAD

**Discussion by bytebebop** [community/community#133961] [New to GitHub] [answered]
In a previous job I saw that when someone made a PR a kind of readme with checkboxes was automatically created. How can I do this on GitHub?

**Comment by oscar-castaneda-dev** [comment, marked as answer]
It's a good question, I'll leave you some steps to follow.

1. create a `.github` directory in the root of the project
2. inside that directory create a file called `PULL_REQUEST_TEMPLATE.md`
3. Inside that markdown file you can put whatever you think is convenient

```
## Description

Some description ...

## Type of change

- [ ] Type 1
- [ ] Type 2
- [ ] Type 3
```

It should be like this:

```

your-repository/
└── .github/
    └── PULL_REQUEST_TEMPLATE.md
```

You can do it directly on github, just make a commit and it should be ready.

**Reply by Kiko191** [reply]
your-repository/
└── .github/
    └── PULL_REQUEST_TEMPLATE.md

**Reply by Kiko191** [reply]
https://github.com/orgs/community/discussions/133961#discussioncomment-10184624

**Comment by Kiko191** [comment]
- [ ]

**Comment by Kiko191** [comment]
I'm not English speaking

**Reply by Kiko191** [reply]
https://github.com/orgs/community/discussions/133961#discussioncomment-10184624

**Comment by Kiko191** [comment]
https://github.com/orgs/community/discussions/133961#discussioncomment-10184624

**Comment by Kiko191** [comment]
Good luck 🤞

**Comment by Kiko191** [comment]
Language of Iran
